### ISSUE TITLE
choco-package-list-backup's automatic backup task doesn't write files

### ISSUE THREAD

**Issue by cal4** [bcurran3/ChocolateyPackages_CCR#202]
On version 2020.04.06 it seems that the choco-package-list-backup task doesn't write files to `C:\Users\Public\ChocolateyPackageListBackup`. The majority of my testing has been when running the task via Run, but this appears to be happening when the task is executed via the schedule as well. This was working at some point, not sure what caused it to break. Worth noting, this is happening on two separate hosts as well.

Troubleshooting steps I've taken:
- Reinstalled choco-package-list-backup without removing the old ChocolateyPackageListBackup directory; no change
- Reinstalled choco-package-list-backup removing the old ChocolateyPackageListBackup directory; no new directory was created

Awesome utility, thanks for sharing!

**Comment by bcurran3** [comment]
Thanks.

I think I see the problem.

I'm testing it overnight.

**Comment by bcurran3** [comment]
I DO see the problem and this will be fixed soon.

**Comment by bcurran3** [comment]
FIXED in today's update, and some new cool stuff... https://github.com/bcurran3/ChocolateyPackages/commit/3c01913be4a48881824bbd8d73587912f509d691

**Comment by cal4** [comment]
@bcurran3 Thanks!

**Comment by bcurran3** [comment]
You're welcome and sorry it took so long. I was very busy juggling multiple projects for work for a few months...now I've got nothing to do and updating my packages. :)
Problem was that I changed the task to run the script via the ChocolateyToolsLocation\blahblah and ChocolateyToolsLocation is a USER environmental variable not a GLOBAL environmental variable. So under SYSTEM, the variable didn't exist; thus failing.

![image](https://user-images.githubusercontent.com/14026600/111374678-9e8ab080-865a-11eb-9b22-8b27464c815d.png)
